authors/apps/authentication/tests/test_signup.py

```diff
@@ -0,0 +1,30 @@
+"""imports relevant to our tests."""
```

Comment:
@engjames This file should not be part of this. You could remove it after testing the pr is working fine.